src/promptflow/promptflow/_utils/multimedia_utils.py

```diff
@@ -193,3 +195,25 @@ def recursive_process(value: Any, process_funcs: dict[type, Callable] = None) ->
     if isinstance(value, dict):
         return {k: recursive_process(v, process_funcs) for k, v in value.items()}
     return value
+
+
+def load_multimedia_data(inputs: dict[str, FlowInputDefinition], line_inputs: dict, base_dir: Path):
+    updated_inputs = dict(line_inputs or {})
+    for key, value in inputs.items():
+        if value.type == ValueType.IMAGE:
+            updated_inputs[key] = create_image(updated_inputs[key], base_dir)
+        elif value.type == ValueType.LIST or value.type == ValueType.OBJECT:
+            updated_inputs[key] = load_multimedia_data_recursively(updated_inputs[key])
+    return updated_inputs
+
+
+def load_multimedia_data_recursively(value: Any):
```

Comment:
```suggestion
def _load_multimedia_data_recursively(value: Any):
```
Will be a internal method?

Reply:
Offline synced, will fix in new PR.